databox/client/lib/DataboxWrapper.ts

```diff
@@ -0,0 +1,60 @@
+import readCommandLineArgs from './utils/readCommandLineArgs';
+import IComponents, { IRunFn } from '../interfaces/IComponents';
+import DataboxInternal from './DataboxInternal';
+import IBasicInput from '@ulixee/databox-interfaces/IBasicInput';
+import IDataboxWrapper from '@ulixee/databox-interfaces/IDataboxWrapper';
+import { setupAutorunBeforeExitHook, attemptAutorun } from './utils/Autorun';
+import IDataboxRunOptions from '@ulixee/databox-interfaces/IDataboxRunOptions';
+
+export default class DataboxWrapper<TInput = IBasicInput, TOutput = any> implements IDataboxWrapper {
+  public static defaultExport: DataboxWrapper;
+  public static disableAutorun = !!process.env.ULX_DATABOX_DISABLE_AUTORUN;
```

Comment:
Should this variable toggle be moved into autorun? You need to explicitly turn off all types of DataboxWrappers if it's setup like this.

Reply:
Good point. Just pushed a commit that moves the env var check into the constructors.